### PR TITLE
Separate processes

### PR DESCRIPTION
**Pull request by frostbyte73**

### CODE REVIEW

**Review by biglittlebigben — APPROVED** [review]
LGTM. One question in line.

File: build/test/Dockerfile [hunk 1]
```diff
@@ -80,9 +80,15 @@ COPY go.sum .
...
 # copy source
+COPY cmd/ cmd/
 COPY pkg/ pkg/
 COPY test/ test/
-COPY build/test/entrypoint.sh .
+COPY version/ version/
+
+# build
+RUN if [ "$TARGETPLATFORM" = "linux/arm64" ]; then GOARCH=arm64; else GOARCH=amd64; fi && \
+    CGO_ENABLED=1 GOOS=linux GOARCH=${GOARCH} GO111MODULE=on go build -a -o /bin/egress ./cmd/server
```
[the remark this answers is not in the document]
Comment (frostbyte73) [reply build/test/Dockerfile:90]: this is for it to run on mac and linux

[the remark this answers is not in the document]
Comment (frostbyte73) [reply build/test/Dockerfile:90]: specifically the new macbook pros

[the remark this answers is not in the document]
Comment (biglittlebigben) [reply build/test/Dockerfile:90]: Thanks

[the remark this answers is not in the document]
Comment (frostbyte73) [reply build/test/Dockerfile:90]: I just realized you were probably asking about the entire thing, not line 89 specifically. It's used for testing the service itself

**Pull request merged**